### PR TITLE
Add delay + walking_time to departure

### PR DESCRIPTION
**Pull request by mrueg**
This is the delay in seconds, we can use this to visualize it on the card prefixed with a + or -

### CODE REVIEW

File: custom_components/berlin_transport/departure.py [hunk 1]
```diff
@@ -43,6 +44,7 @@ def from_dict(cls, source):
                 source.get("currentTripPosition", {}).get("longitude") or 0.0,
             ],
             cancelled=source.get("cancelled", False),
+            delay=source.get("delay", 0),
```
[the remark this answers is not in the document]
Comment (mrueg) [reply custom_components/berlin_transport/departure.py:47]: I understand your point. How would you do that with an int, where the None type does not exist?

[the remark this answers is not in the document]
Comment (Cornelicorn) [reply custom_components/berlin_transport/departure.py:47]: Same as the other definitions in the dataclass, I added a suggestion above.

[the remark this answers is not in the document]
Comment (Cornelicorn) [reply custom_components/berlin_transport/departure.py:47]:
```suggestion
            delay=source.get("delay", None),
```

File: custom_components/berlin_transport/departure.py [hunk 2]
```diff
@@ -20,6 +20,7 @@ class Departure:
     fallback_color: str | None = None
     location: tuple[float, float] | None = None
     cancelled: bool = False
+    delay: int = 0
```
Comment (Cornelicorn) [inline custom_components/berlin_transport/departure.py:23]:
```suggestion
    delay: int | None = None
```

File: custom_components/berlin_transport/departure.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mrueg) [reply custom_components/berlin_transport/departure.py]: @Cornelicorn thanks for the suggestion, mypy failed when I tried it.

**Review by vas3k — APPROVED** [review]

**Pull request merged**